resources/ubuntu-build.sh

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
+
+set -x
+set -e
+
+USRSCTPPATH=usrsctp
+JNIPATH=src/main/native
+JAVAHPATH=target/native/javah
+OUTPATH=target/classes/lib
+RESOURCESPATH=src/main/resources
+
+if [ "$#" -ne 3 ]; then
+    echo "Usage: $0 <ARCH> <JAVA_VERSION> <DIR>"
+    echo "  ARCH: Architecture to build for (amd64, arm64, ppc64el)"
+    echo "  JAVA_VERSION: Java version (11)"
+    echo "  DIR: sctp4j project directory"
+    exit 1
+fi
+
+DEBARCH=$1
+JAVA_VERSION=$2
+DIR="$3"
+
+cd "$DIR"
+
+if [ \! -r $JAVAHPATH/org_jitsi_modified_sctp4j_SctpJni.h ]; then
+    echo "$JAVAHPATH/org_jitsi_modified_sctp4j_SctpJni.h not found: did you run mvn compile?"
+    exit 1
+fi
+
+case $DEBARCH in
+    amd64)
+	GNUARCH=x86_64
+	JNAARCH=x86-64
+	;;
+    arm64)
+	GNUARCH=aarch64
+	JNAARCH=aarch64
+	;;
+    ppc64el)
+	GNUARCH=powerpc64le
+	JNAARCH=ppc64le
+	;;
+    *)
+	echo "ERROR: Unsupported arch $DEBARCH!"
+	exit 1
+	;;
+esac
+
+NATIVEDEBARCH=$(dpkg --print-architecture)
+
+if [ $DEBARCH != $NATIVEDEBARCH ]
+then
+    HOST=$GNUARCH-linux-gnu
+    CONFIGURE_ARGS="--host $HOST"
+    export CC=$HOST-gcc
+else
+    export CC=gcc
+fi
+
+NCPU=$(grep -c processor /proc/cpuinfo)
```

Comment:
The most portable way to write this for Linux is `getconf _NPROCESSORS_ONLN` -- it takes into account online and offline CPUs.

Reply:
The typical Makefile argument is `$(nproc)`